### PR TITLE
Add editcontent checkbox label left, right, left+right

### PR DESCRIPTION
**Pull request by rarila**
As discussed in  #4616 .

@sbonardt , @bobdenotter : May I ask one of you to add that new behavior to the documentation?  :pray:

### CODE REVIEW

File: app/view/twig/_macro/_buic.twig [hunk 1]
```diff
@@ -5,6 +5,7 @@
  #    class      ''         Classname added to the component container.
  #    disabled   false      Input is disabled.
  #    id         name       Id of the checkbox.
+ #    info       ''         Infotext to be displayed i there's a label.
```
Comment (bobdenotter) [inline app/view/twig/_macro/_buic.twig]: typo in "if"

**Comment by GawainLynch** [comment]
:koala: :heavy_check_mark: 
:de: :heavy_check_mark: 
:us: :heavy_check_mark:

**Comment by rarila** [comment]
@GawainLynch What's that? [Fun with flags](http://vignette2.wikia.nocookie.net/bigbangtheory/images/b/b9/Funwithflags_zpsf57e3cf1.jpg/revision/latest?cb=20141216003140)?

**Comment by GawainLynch** [comment]
@rarila something like… thaaaaat…

**Comment by SahAssar** [comment]
Hey! Vexillology is no laughing matter!

**Pull request merged**